### PR TITLE
BRICK-801: install-bricks.sh

### PR DESCRIPTION
**Pull request by mordr**

### CODE REVIEW

**Comment by mordr** [comment]
placeholder for `install-bricks.sh`

File: cl.conf [hunk 1]
```diff
@@ -36,6 +36,20 @@ systemd:
         RemainAfterExit=true
         User=core
 
+        [Install]
+        WantedBy=multi-user.target
```
Comment (ahawtho) [inline cl.conf:40]: Why did you add this?

Comment (mordr) [reply cl.conf:40]: It's already present in `BRICK/develop` branch, not sure why it shows as new addition

```
    - name: install-minikube.service
      enable: true
      contents: |
        [Unit]
        Description=Install minikube
        ConditionPathExists=!/usr/local/bin/localkube

        [Service]
        Type=oneshot
        ExecStart=/opt/minikube/install-minikube.sh
        RemainAfterExit=true
        User=core

        [Install]
        WantedBy=multi-user.target
```

* git log shows it correctly
```
| diff --git a/cl.conf b/cl.conf
| index ad99030..df9e3d3 100644
| --- a/cl.conf
| +++ b/cl.conf
| @@ -38,6 +38,20 @@ systemd:
|
|          [Install]
|          WantedBy=multi-user.target
| +    - name: install-bricks.service
| +      enable: true
| +      contents: |
| +        [Unit]
| +        Description=Install bricks
| +
| +        [Service]
| +        Type=oneshot
| +        ExecStart=/opt/bricks/install-bricks.sh
| +        RemainAfterExit=true
| +        User=core
| +
| +        [Install]
| +        WantedBy=multi-user.target
```

**Comment by ahawtho** [comment]
Ok. I was surprised that it showed as an addition because I should have
added it already, but I also knew that it was running at boot-time, so if
it wasn't already there, I thought it might not have been necessary after
all.
